instrumentation/rocketmq-client/src/test/java/brave/rocketmq/client/ITRocketMQTracing.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package brave.rocketmq.client;
+
+import brave.handler.MutableSpan;
+import brave.messaging.MessagingRequest;
+import brave.messaging.MessagingTracing;
+import brave.sampler.Sampler;
+import brave.sampler.SamplerFunction;
+import brave.sampler.SamplerFunctions;
+import brave.test.ITRemote;
+import brave.test.IntegrationTestSpanHandler;
+import org.apache.rocketmq.client.consumer.DefaultMQPushConsumer;
+import org.apache.rocketmq.client.consumer.listener.ConsumeConcurrentlyStatus;
+import org.apache.rocketmq.client.consumer.listener.ConsumeOrderlyStatus;
+import org.apache.rocketmq.client.consumer.listener.MessageListenerConcurrently;
+import org.apache.rocketmq.client.consumer.listener.MessageListenerOrderly;
+import org.apache.rocketmq.client.producer.DefaultMQProducer;
+import org.apache.rocketmq.client.producer.SendCallback;
+import org.apache.rocketmq.client.producer.SendResult;
+import org.apache.rocketmq.common.message.Message;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static brave.Span.Kind.CONSUMER;
+import static brave.Span.Kind.PRODUCER;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Tag("docker")
+@Testcontainers(disabledWithoutDocker = true)
+@Timeout(240)
+class ITRocketMQTracing extends ITRemote {
+  static final Logger LOGGER = LoggerFactory.getLogger(ITRocketMQTracing.class);
```

Comment:
I believe we shouldn't be using SLF4J directly but `java.util.logging.Logger`:

```suggestion
import java.util.logging.Logger;

  static final Logger LOGGER = Logger.getLogger(ITRocketMQTracing.class.getName());
```